modules/bitgo/test/v2/unit/coins/near.ts

```diff
@@ -554,4 +553,163 @@ describe('NEAR:', function () {
       }
     });
   });
+
+  describe('Parse Transactions:', () => {
+    const amount = '10000000000000000000000000000000';
+    const gas = '125000000000000';
+
+    it('should parse an unsigned transfer transaction', async function () {
+      const parsedTransaction = await basecoin.parseTransaction({
+        txPrebuild: {
+          txHex: rawTx.transfer.unsigned,
+        },
+        feeInfo: {
+          fee: '5000',
+        },
+      });
+
+      parsedTransaction.should.deepEqual({
+        inputs: [{
+          address: '9f7b0675db59d19b4bd9c8c72eaabba75a9863d02b30115b8b3c3ca5c20f0254',
+          amount: '1000000000000000000000000',
+        }],
+        outputs: [
+          {
+            address: '9f7b0675db59d19b4bd9c8c72eaabba75a9863d02b30115b8b3c3ca5c20f0254',
+            amount: '1000000000000000000000000',
+          },
+        ],
+      });
+    });
+
+    it('should parse a signed transfer transaction', async function () {
+      const parsedTransaction = await basecoin.parseTransaction({
+        txPrebuild: {
+          txHex: rawTx.transfer.signed,
+        },
+        feeInfo: {
+          fee: '',
+        },
+      });
+
+      parsedTransaction.should.deepEqual({
+        inputs: [{
+          address: '9f7b0675db59d19b4bd9c8c72eaabba75a9863d02b30115b8b3c3ca5c20f0254',
+          amount: '1000000000000000000000000',
```

Comment:
then create a constant for these variables and lets add a coment about how much near is

Reply:
Done!